packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -1,4 +1,36 @@
-import { html } from '@microsoft/fast-element';
+import { html, repeat } from '@microsoft/fast-element';
 import type { WaferMap } from '.';
+import type { WaferMapDie } from './types';
 
-export const template = html<WaferMap>``;
+const notReadyTemplate = html<WaferMap>` <div>Loading....</div> `;
+
+const waferTemplate = html<WaferMap>`
+    <div>
+        <ul>
+            ${repeat(
+        x => x.dies,
+        html`
```

Comment:
instead of typing the arrow function, you should type the `html` template tag

```ts
html<WaferMapDie>`blah blah ${x => `dieX: ${x.x}`} blah`;
```

Reply:
applied